relay/src/main.rs

```diff
@@ -38,7 +38,7 @@
 //!  - [`relay-cogs`]: Break down the cost of Relay by its features.
 //!  - [`relay-common`]: Common utilities and crate re-exports.
 //!  - [`relay-config`]: Static configuration for the CLI and server.
-//!  - [`relay-conventions`]: Attribute definitions extracted from [`sentry-conventions`](https://github.com/getsentry/sentry-conventions).
+//!  - `relay-conventions`: Attribute definitions extracted from [`sentry-conventions`](https://github.com/getsentry/sentry-conventions).
```

Comment:
This is a bit unfortunate. I assume it has to do with the fact that `relay-conventions` has a build script, but I'm not sure.